docs/DeveloperGuide.md

```diff
@@ -700,35 +700,110 @@ Given below are instructions to test the app manually.
+
+#### Deleting a student
+1. Prerequisites: There must be at least 1 student currently displayed on the Student view list.
+1. Test case: `delete_student 1`<br>
+   Expected: First student deleted from the list. Details of the deleted student shown in the status message.
+1. Incorrect commands to try: `delete_student 0` or `delete_student x` (where x is larger than the size of the student list) <br>
+   Expected: No student is deleted. Error details shown in the status message.
+
+### Managing Sessions
+
+#### Adding a session
+1. Prerequisites: There must be at least 1 student named `Alex Yeoh` currently displayed on the Student view list.
+1. Test case: `add_session n/Alex Yeoh d/2021-04-10 t/12:00 k/120 s/Math f/100`<br>
+   Expected: The session with the fields will be added to the Session list view under the Student's name. The session will have a light blue "I" labelled
+   at the bottom right of the session card. If this session falls within 3 days from the current day (today, tomorrow, day after tomorrow),
+   this session will be displayed in the reminder list view on the Home tab.
+1. Incorrect commands to try: `add_session` or `add_session n/Alex Yeoh d/10-04-2021 t/12:00 k/120 s/Math f/100` <br>
+   Expected: No session is added. Error details shown in the status message.
+
+#### Adding a recurring session
+1. Prerequisites: There must be at least 1 student named `Alex Yeoh` currently displayed on the Student view list.
+1. Test case: `add_rec_session n/Alex Yeoh d/2021-04-10 e/2021-06-26 b/7 t/12:00 k/120 s/Math f/100`<br>
+   Expected: The session with the fields will be added to the Session list view under the Student's name. The session will have a orange "R" labelled
+   at the bottom right of the session card. If there is a session that falls within 3 days from the current day (today, tomorrow, day after tomorrow),
+   this session will be displayed in the reminder list view on the Home tab.
+1. Incorrect commands to try: `add_rec_session` or `add_rec_session n/Alex Yeoh d/2021-04-10 e/2021-06-28 b/7 t/12:00 k/120 s/Math f/100` <br>
+   Expected: No recurring session is added. Error details shown in the status message.
+
+#### Deleting a session or recurring session
+1. Prerequisites: There must be at least 1 session or recurring session for the student named `Alex Yeoh`.
+1. Test case: `delete_session n/Alex Yeoh i/1`<br>
+   Expected: First session from Alex Yeoh will be deleted from the list. Details of the deleted session shown in the status message.
+1. Incorrect commands to try: `delete_session n/anonymous i/1` (name does not exist in the student list) or
+   `delete_session n/Alex Yeoh i/x` (where x is larger than the size of the session list for `Alex Yeoh`) <br>
+   Expected: No session is deleted. Error details shown in the status message.
+
+#### Deleting a single session from a recurring session
+1. Prerequisites: There must be a recurring session in the session list for the student named `Alex Yeoh`.
```

Comment:
May want to include that session date must be a date belonging to the recurring session! 😃